src/views/clusteroverview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesList/components/LoadBalanceSection/LoadBalanceSection.tsx

```
@@ -1,33 +1,44 @@
-import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Button, Split, SplitItem } from '@patternfly/react-core';
 
 import IconSkeleton from '../icons/IconSkeleton/IconSkeleton';
 
 import './LoadBalanceSection.scss';
 
 const LoadBalanceSection: FC = () => {
   const { t } = useKubevirtTranslation();
+  const navigate = useNavigate();
   const { operatorDetailsMap, operatorResourcesLoaded } = useVirtualizationFeaturesContext();
   const { installState, operatorHubURL } = operatorDetailsMap?.[DESCHEDULER_OPERATOR_NAME];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)